R/dist.R

```diff
@@ -821,57 +867,242 @@ tune_inv_gamma <- function(lower = 2, upper = 21) {
 #' @author Sam Abbott
 #' @export
 dist_spec <- function(mean, sd = 0, mean_sd = 0, sd_sd = 0,
-                      dist = c("lognormal", "gamma"), max = NULL,
-                      fixed = FALSE) {
-  dist <- match.arg(dist)
+                      dist = c("lognormal", "gamma"), max,
```

Comment:
we may as well call this arg distribution (or change to family) as we aren't really getting much by shortening it

Reply:
Resolved in https://github.com/epiforecasts/EpiNow2/pull/363/commits/c99a07f178afda437b19c82b7090439b57e8eff9